python/vyos/ifconfig/bond.py

```diff
@@ -54,6 +55,12 @@ class BondIf(Interface):
             'validate': lambda v: assert_list(v, ['slow', 'fast']),
             'location': '/sys/class/net/{ifname}/bonding/lacp_rate',
         },
+
```

Comment:
Please drop the empty lines before and after to match the current structure.

Reply:
done!